src/Abstractions/NexusMods.Abstractions.Cli/RendererExtensions.cs

```diff
@@ -64,6 +64,7 @@ public static async Task<T> WithProgress<T>(this IRenderer renderer, Cancellatio
     /// <param name="args"></param>
     public static async Task Error(this IRenderer renderer, Exception ex, string template, params object[] args)
     {
-        await renderer.Text(template + "\n\nTraceback: {ex} \n ", args);
+        await renderer.Text(template, args);
```

Comment:
It's a cold path, so not really a big deal, but latency here would be lower if it was still 1 render message across IPC.

Reply:
I'm not really concerned as this is only used when throwing an exception, and the templating here was really strange otherwise, as we have to concat the arguments and the template